docs/source/install/docker-compose.rst

```diff
@@ -0,0 +1,275 @@
+
+2) Define an harvest job configuration as specified in :doc:`Harvest Job Configuration<../user/harvest_job_configuration>`
+
+3) Run the harvest command:
+
+    ./pds-batch-loader.sh  <harvest job confg file>
+
+This is using docker compose in the background so no need to install anything more.
+
+4) Wait for the following log messages in the terminal. ::
+
+    docker-registry-loader-1 exited with code 0
+
+5) Visit the http://localhost:8080/swagger-ui.html#!/collections/getCollection of the Registry API.
+
+6) Click on the **Try it out!** button to see the Response Body.
+
+
+Clean-up the Deployment
+------------------------
+
+The docker containers deployed above can be easily uninstalled and cleaned-up using the following
+commands::
+
+    docker compose --profile=pds-core-registry down
+
+
+Core registry with Scalable Harvest
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As explained above, the Scalable Harvest is suitable to process larger data sets of more than 10,000 PDS4 labels.
+
+You can execute the following instructions to deploy the server-side components of Registry with Scalable Harvest and load some data.
+
+.. note::
+  Docker Compose running on a single host, the scalability of this tool is not at its full potential yet, we will propose in the future a production ready deployment procedure (e.g. using kubernetes) to actually scale the ingestion.
+
+
+Deploying the Sever-side Components of Scalable Harvest
+--------------------------------------------------------
+
+1) Open a new terminal and change the current working directory to the ``<REGISTRY_ROOT>/docker`` directory.
+
+2) Start opensearch and the PDS search API as follows. ::
+
+    docker compose --profile=pds-core-registry  up
+
+3) Wait for the following log messages in the terminal. ::
+
+    docker-registry-api-1        |   .   ____          _            __ _ _
+    docker-registry-api-1        |  /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
+    docker-registry-api-1        | ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
+    docker-registry-api-1        |  \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
+    docker-registry-api-1        |   '  |____| .__|_| |_|_| |_\__, | / / / /
+    docker-registry-api-1        |  =========|_|==============|___/=/_/_/_/
+    docker-registry-api-1        |  :: Spring Boot ::        (v2.3.1.RELEASE)
+    docker-registry-api-1        |
+
+4) Start the `pds-loader-services` components as follows to start Scalable Harvest components. ::
+
+    docker compose --profile=pds-loader-services up
+
+Run Scalable Harvest Service client to load some PDS4 products
+-----------------------------------------------------------
+
+1) Open a new terminal and change the current working directory to the ``<REGISTRY_ROOT>/docker/scripts/`` directory.
+
+2) Define an harvest job configuration as specified in :doc:`Harvest Job Configuration<../user/harvest_job_configuration>`
```

Comment:
Minor typo: "an harvest" should be "a harvest"

Reply:
Done